app/config.py

```diff
@@ -66,3 +66,9 @@
     # Our most-read Collections -> QA duplicate of this slate
     "0f322865-64e6-472d-8147-b3d6637a7d67": "b70d65c6-9171-40bf-bddb-5a60d42dd03f",
 }
+
+# This will redirect to the slate lineup value if the slate lineup id raises an exception
+fallback_slate_lineup = {
```

Comment:
```suggestion
# This will redirect to the slate lineup value if the slate lineup id raises a PersonalizationError exception
personalization_fallback_slate_lineup = {
```
Rename this because we're currently only catching `PersonalizationError`. It's not a generic fallback as we implemented it.